src/helpers.ts

```diff
@@ -321,6 +321,23 @@ export function wrapFnWithLocation<A extends unknown[], R>(
   };
 }
 
+// Safely parse ND JSON (Newline delimitted JSON) chunks
+export function safeNDJSONParse(chunks: string[]) {
+  // chunks may not be at proper newline boundaries, so we make sure everything is split
+  // on proper newlines
+  chunks = Array.isArray(chunks) ? chunks : [chunks];
+  const lines = chunks.join('\n').split(/\r?\n/);
+  return lines
+    .filter(l => l.match(/\S/)) // remove blank lines
+    .map(data => {
```

Comment:
```suggestion
    .map(line => {
```

I think naming this `line` instead of `data` will make the code more readable.

Reply:
Sure can you push the change along with the merge change https://github.com/elastic/synthetics/pull/576#discussion_r944535091

Reply:
++